### PR TITLE
D8CORE-106: Add Media with Caption paragraph.

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW

# Summary
- Adds a Media with a Caption paragraph.

# Review By (Date)
- January 15th

# Urgency
- low

# Steps to Test

1. See https://github.com/SU-SWS/acsf-cardinalsites/pull/51

# Affected Projects or Products
- D8CORE
- Greg

# Associated Issues and/or People
- D8CORE-106
- @imonroe FYI

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by pookmish** [comment]
no caption needed

**Comment by sherakama** [comment]
@pookmish ready for review again.

File: composer.json [hunk 1]
```diff
@@ -96,15 +96,15 @@
         "drupal/webform": "^5.4",
         "drupal/xmlsitemap": "~1.0@alpha",
         "ext-imagick": "*",
-        "su-sws/jumpstart_ui": "^8.1.0",
+        "su-sws/jumpstart_ui": "dev-D8CORE-106",
```
Comment (pookmish) [inline composer.json]:
```suggestion
        "su-sws/jumpstart_ui": "^8.1.0",
```

File: composer.json [hunk 2]
```diff
@@ -96,15 +96,15 @@
...
         "su-sws/nobots": "^8.1.0",
         "su-sws/react_paragraphs": "^8.1.0@alpha",
         "su-sws/stanford_basic": "^8.4.0",
         "su-sws/stanford_date_formats": "^8.1.0",
         "su-sws/stanford_fields": "^8.1.0",
         "su-sws/stanford_image_styles": "^8.1.0",
         "su-sws/stanford_ssp": "^8.1.1",
-        "su-sws/stanford_text_editor": "^8.1.0"
+        "su-sws/stanford_text_editor": "dev-D8CORE-106"
```
Comment (pookmish) [inline composer.json]:
```suggestion
        "su-sws/stanford_text_editor": "^8.1.0"
```

**Comment by sherakama** [comment]
Thank you!

**Comment by pookmish** [comment]
Once the green lights. go ahead and merge

**Comment by sherakama** [comment]
Green lights won't turn on as composer now points to earlier tags that don't have the new functionality in them and the behat tests should fail. This is a gap in our process as the composer version constraints are technically incorrect at this point as they do require dev branches.

**Comment by pookmish** [comment]
ahh well thats annoying. we could cut tags in those modules to allow these to pass. :man_shrugging:

**Comment by sherakama** [comment]
We are doing that Friday. How about switching to the dev branches as this will come up again this week I am sure.

**Comment by pookmish** [comment]
fine by me :+1:

**Comment by sherakama** [comment]
We changed our minds and decided to tag instead.

**Comment by sherakama** [comment]
Ok, re-ran tests with the new tags. We're gtg now.

**Comment by pookmish** [comment]
push the button

**Pull request merged**